internal/repository/delete_task_by_id.go

```diff
@@ -0,0 +1,30 @@
+package repository
+
+import (
+	"database/sql"
+
+	"github.com/msmkdenis/go-task-tracker/internal/model"
+)
+
+func (t *SQLiteTaskRepository) DeleteByID(id int64) error {
+	res, err := t.db.Exec(
+		`
+		delete from scheduler
+		where id = :id
+		`,
+		sql.Named("id", id))
+	if err != nil {
+		return err
```

Comment:
Оборачивай ошибки, а то потом не понятно будет откуда они взять

```suggestion
		return fmt.Errorf("DeleteByID: error in exec: %w", err)
```